### DISCUSSION TITLE
Immich - Alterar local de armazenamento upload

### DISCUSSION THREAD

**Discussion by leiteporto** [immich-app/immich#13033] [Q&A]
Olá a todos. Estou conhecendo o Server Immich agora e já acho espetacular, mas não tenho muito conhecimento nele, ainda.
Pretendo montar unidades de armazenamento dedicadas e gostaria de saber como faço para mudar o local de armazenamento de arquivos de upload. O que preciso fazer? Se alguém puder me ajudar, agradeço desde já.

**Comment by alextran1502** [comment]
Hello, let's keep the communication in English so it is easy for all the contributors and other users 😛

**Reply by leiteporto** [reply]
Hello everybody. I'm getting to know the Immich Server now and I think it's spectacular, but I don't have much knowledge of it yet.
I intend to set up dedicated storage units and would like to know how to change the storage location for upload files. What do I need to do? If anyone can help me, I would appreciate it in advance.
